desktop/main.dev.ts

```diff
@@ -242,19 +244,33 @@ const createWindow = async () => {
     shell.openExternal(url);
   });
 
-  const res = await fetch(DISCOVERY_URL);
-  const initialConfig = (await res.json())[0];
   const savedNetId = StoreService.get('netSettings.netId');
-  const cleanStart = savedNetId !== initialConfig.netID;
-  const configFilePath = path.resolve(app.getAppPath(), process.env.NODE_ENV === 'development' ? './' : '../../config', 'config.json');
+  const configFilePath = path.resolve(app.getPath('userData'), 'node-config.json');
+  let netId;
+  let initialConfig;
+  let netConfig;
+  let isDevNet = false;
+  if (process.env.NODE_ENV === 'development' && process.env.DEV_NET_URL) {
+    const devConfig = await fetch(process.env.DEV_NET_URL);
+    netConfig = await devConfig.json();
+    netId = netConfig.p2p['network-id'];
+    isDevNet = true;
+  } else {
+    const res = await fetch(DISCOVERY_URL);
```

Comment:
```
initialConfig = (await fetch(DISCOVERY_URL).then((res) => res.json()))[0];
netConfig = await fetch(initialConfig.conf).then((res) => res.json());
```
instead of dumb names like `res`, `res2` and additional `if` below